internal/page/app_data.go

```diff
@@ -26,6 +26,7 @@ type AppData struct {
 	AttorneyUID       actoruid.UID
 	LoginSessionEmail string
 	SupporterData     *SupporterData
+	PublicURL         string
```

Comment:
I know we're trying to avoid stuffing too much in `AppData` but this really feels like it belongs here.

If we merge this I'll make a follow-up PR to replace function args with `AppData`.

Reply:
we've survived this long with passing it through, so why not do that again?

Reply:
`AppData` is a big bag of arguments we pass to every handler, so I think the bar for adding things should be pretty high

Reply:
OK, that's a fair point